exercicios/exercicio2.js

```diff
@@ -0,0 +1,5 @@
+// 2. Liste a tabuada do 6 (do 1 ao 10)
+
+const tabuadaSeis = () => {for (i = 1; i <= 10; i++) {console.log(i*6)}}
```

Comment:
Quando a gente usar o `for`, o comum é quebrar em várias linhas mesmo ao invés de fazer inline. (:

Reply:
Anna, obrigada pelo code review! Tô sempre no github mas não tinha visto a notificação pelo pc, apenas no email! Eu vou ver tudo, tudinho, pelo pc mesmo. Você é demais!